stories/assets/js/sidebar.js

```diff
@@ -1,37 +1,37 @@
 export function sidebarNav() {
-  let $accordionNav = $('.accordion-navbar li');
-  let $accordionPanel = $('.accordion-navbar .accordion__panel');
+  let $accordionNav = $('.sidebar-accordion li');
+  let $accordionPanel = $('.sidebar-accordion .accordion__panel');
   $accordionNav.each((index, element) => {
     if ($(element).find('.accordion__panel').length) {
-      $(element).find('span').first().off('click')
-        .click(function (e) {
-          e.preventDefault();
-          e.stopPropagation();
-          const duration = '30';
-          let $activeLi = $(this).closest('li');
-          let $activePanel = $activeLi.find('.accordion__panel');
-          if ($activeLi.hasClass('accordion--active') && $activePanel.is(':visible')) {
-            $activePanel.slideUp(duration);
-            $activeLi.removeClass('accordion--active');
-          } else {
-            $accordionNav.removeClass('accordion--active');
-            $accordionPanel.slideUp(duration);
-            $activePanel.slideDown(duration);
-            $activeLi.addClass('accordion--active');
-          }
-        });
+      $(element).find('span').first().off('click keypress').on('click keypress', function (e) {
+        console.log("enter");
+        e.preventDefault();
+        e.stopPropagation();
+        const duration = '30';
+        let $activeLi = $(this).closest('li');
+        let $activePanel = $activeLi.find('.accordion__panel');
+        if ($activeLi.hasClass('accordion--active') && $activePanel.is(':visible')) {
+          $activePanel.slideUp(duration);
+          $activeLi.removeClass('accordion--active');
+        } else {
+          $accordionNav.removeClass('accordion--active');
+          $accordionPanel.slideUp(duration);
+          $activePanel.slideDown(duration);
+          $activeLi.addClass('accordion--active');
+        }
+      })
```

Comment:
@pallavikadam-srijan Isn't this already defined in `accordion.js`? Why are we not importing that js?